app/assets/javascripts/application.js

```diff
@@ -21,3 +21,18 @@ $(function(){ $(document).foundation(); });
 $(document).on('change', '.autosubmitme', function() {
   $(this).parents('form').submit();
 });
+
+$(document).on('click', '.suites-fields .check-all', function() {
+  set_checkbox_of_parent(this, '.suites-fields', true);
+  return false; // prevent scrolling behavior
```

Comment:
Apparently returning false in the other function doesn't carry through to the callback (as I would expect in Ruby).